package.json

```diff
@@ -54,8 +54,8 @@
         "test": "eslint src && nyc ava",
         "clean": "rm -rf lib",
         "coverage": "nyc report --reporter=lcov > coverage.lcov && codecov",
-        "flow-strip": "flow-remove-types -x .js --pretty --sourcemaps inline src/ --out-dir lib/",
-        "build": "npm run clean && npm run flow-strip && cp src/index.d.ts lib/index.d.ts && cp src/index.js.flow lib/index.js.flow",
+        "babel-flow": "babel src --out-dir lib --presets=es2015,flow",
```

Comment:
It’s probaly best to use babel-preset-env that way it supports current node instead of a specific version.